app/models/miq_schedule_worker/jobs.rb

```diff
@@ -121,6 +121,14 @@ def miq_report_result_purge_timer
     queue_work(:class_name => "MiqReportResult", :method_name => "purge_timer", :zone => nil)
   end
 
+  def archived_entities_purge_timer
```

Comment:
@jrafanie up until now, `Job` inserts a single work record and the model's `Purge` coordinates multiple records.

But this is nice that it reduces the number of settings timers